docs/share.md

```diff
@@ -12,19 +12,28 @@ You can share the data your connector accesses by connecting to the data, creati
 
 Or, if you prefer to have your users test the connector, you can place the connector files in the directories and then run the commands as follows:
 
-**Tableau Desktop:** This is the same as developing the connector.
-Place the files into a subdirectory you create, and then use the -DConnectPluginsPath command line argument.  For example, place your plugin files in `C:\tableau_connectors\myconnector` and then start Tableau:
+**Tableau Desktop:** 
+
+1. Place the files into a subdirectory you create.
+1. Use the `-DConnectPluginsPath` command line argument.  
```

Comment:
Should be 2, not 1.

Reply:
That's the markdown numbering scheme. It displays as 2.